File: Tema 4/Ejercicios/Ejercicios Listas/Ejercicios Listas 02/Ejercicio 3.py
cuenta=int(input("Dime cuantos valores quieres introducir: "))
cuen=0
lista=[]
while cuen < cuenta:
    cuen+=1
    if cuen > 0:
        palabra=input("Dime la asignatura: ")
        nota=input("Dime tu nota: ")
        lista2=[nota]
        lista.clear()
        lista.append(palabra)
    print("En",lista,"Tengo un",lista2)

File: Java/Ejercicios Repaso java 1/Ejercicios repaso 1/src/Ejercicio_3.java
import java.util.ArrayList;
import java.util.HashSet;
import java.util.Scanner;
import java.util.Set;

public class Ejercicio_3 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("¿Cuantos numeros quieres introducir? ");
        int introduccion = scanner.nextInt();
        ArrayList<Integer> lista = new ArrayList<>();
        for (int i = 0; i < introduccion; i++) {
            System.out.println("Dime el numero ");
            int numero = scanner.nextInt();
            lista.add(numero);
        }
        Set nodupli = new HashSet<>(lista);
        lista.clear();
        lista.addAll(nodupli);
        System.out.println(lista);
    }
}

